src/bentoml/server.py

```diff
@@ -93,14 +93,15 @@ def __init__(
 
     def _create_startmanager(server):  # type: ignore # not calling self self
         class _StartManager:
-            def __init__(self, blocking: bool = False):
+            def __init__(self, blocking: bool = False, **kwargs: t.Any):
                 logger.info(f"starting server with arguments: {server.args}")
 
                 server.process = subprocess.Popen(
                     server.args,
                     stdout=subprocess.PIPE,
                     stderr=subprocess.PIPE,
                     stdin=subprocess.PIPE,
+                    **kwargs,
```

Comment:
I think we probably should support specific arguments, was there something you thought should be supported?

Maybe we should have something to configure the standard in/outs.